test/data/v3/basic/tests/deployment_test.yaml

```diff
@@ -62,6 +62,9 @@ tests:
           of: extensions/v1beta1
       - hasDocuments:
           count: 2
+      - hasDocuments:
+          count: 1
+          filterAware: true
```

Comment:
shell we try to find a slightly better naming?

`selectorAware` or similar? Or we planning for this filter to do more then that?

Reply:
shell we try to find a slightly better naming?

`selectorAware|filterSelector` or similar? Or we planning for this filter to do more then current functionality?